BWEnv/src/controller.cc

```diff
@@ -71,12 +81,18 @@ bool Controller::connect_server() {
 
   assert(this->zmq_server->server_sock_connected);
 
-  output_log.open(
-      config_->log_path + std::to_string(this->zmq_server->getPort()) + ".txt");
-  Utils::bwlog(
-      output_log,
-      "Successfully connected to proxy client on port %d.",
-      this->zmq_server->getPort());
+  output_log.open(log_path);
+  if (config_->file_socket.size() > 0) {
+    Utils::bwlog(
+        output_log,
+        "Successfully connected to proxy client on socket at %sd.",
```

Comment:
TBH i have no clue why this says "proxy client". Can you change this to "client"?